virtio-queue/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Upcoming
 
+## Changed
+
+- Use `RawDescriptor` to represent the memory layout of the split and packed descriptor in virtio-queue/desc.
+- Move the split descriptor to the virtio-queue/desc.
+
+## Added
+
+- Add packed descriptor in virtio-queue
```

Comment:
The packed descriptor is not used in tests at the moment, do you have ideas/thoughts on how to do that?

Reply:
(To clarify, I meant in actual usage from an interface "consumer" crate, besides the unit tests that exist right now)

Reply:
I think it's hard to test from an interface "consumer" crate, except we add the packed vring support. Do you think that it needs to be submitted in this pr?